images_test.go

```diff
@@ -74,18 +75,110 @@ func TestImages_RetrievePageByNumber(t *testing.T) {
 	checkCurrentPage(t, resp, 2)
 }
 
```

Comment:
@aybabtme any value to adding tests for ListDistribution, ListApplication, ListUser?